2022-08-05 14:31:22,888 INFO    MainThread:12868 [wandb_setup.py:_flush():76] Configure stats pid to 12868
2022-08-05 14:31:22,888 INFO    MainThread:12868 [wandb_setup.py:_flush():76] Loading settings from C:\Users\psiml8\.config\wandb\settings
2022-08-05 14:31:22,888 INFO    MainThread:12868 [wandb_setup.py:_flush():76] Loading settings from c:\Users\psiml8\CausalWorld\scripts\wandb\settings
2022-08-05 14:31:22,888 INFO    MainThread:12868 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-08-05 14:31:22,888 INFO    MainThread:12868 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-08-05 14:31:22,888 INFO    MainThread:12868 [wandb_init.py:_log_setup():461] Logging user logs to c:\Users\psiml8\CausalWorld\scripts\wandb\run-20220805_143122-16s1mhi1\logs\debug.log
2022-08-05 14:31:22,889 INFO    MainThread:12868 [wandb_init.py:_log_setup():462] Logging internal logs to c:\Users\psiml8\CausalWorld\scripts\wandb\run-20220805_143122-16s1mhi1\logs\debug-internal.log
2022-08-05 14:31:22,889 INFO    MainThread:12868 [wandb_init.py:_jupyter_setup():411] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000002144CFD0688>
2022-08-05 14:31:22,889 INFO    MainThread:12868 [wandb_init.py:init():495] calling init triggers
2022-08-05 14:31:22,889 INFO    MainThread:12868 [wandb_init.py:init():499] wandb.init called with sweep_config: {}
config: {}
2022-08-05 14:31:22,889 INFO    MainThread:12868 [wandb_init.py:init():548] starting backend
2022-08-05 14:31:22,890 INFO    MainThread:12868 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-08-05 14:31:22,896 INFO    MainThread:12868 [backend.py:ensure_launched():217] starting backend process...
2022-08-05 14:31:23,007 INFO    MainThread:12868 [backend.py:ensure_launched():222] started backend process with pid: 13000
2022-08-05 14:31:23,008 INFO    MainThread:12868 [wandb_init.py:init():558] backend started and connected
2022-08-05 14:31:23,091 INFO    MainThread:12868 [wandb_run.py:_label_probe_notebook():1084] probe notebook
2022-08-05 14:31:23,092 INFO    MainThread:12868 [wandb_run.py:_label_probe_notebook():1094] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-08-05 14:31:23,092 INFO    MainThread:12868 [wandb_init.py:init():634] updated telemetry
2022-08-05 14:31:23,157 INFO    MainThread:12868 [wandb_init.py:init():666] communicating run to backend with 30 second timeout
2022-08-05 14:31:25,033 INFO    MainThread:12868 [wandb_run.py:_on_init():1919] communicating current version
2022-08-05 14:31:25,091 INFO    MainThread:12868 [wandb_run.py:_on_init():1923] got version response upgrade_message: "wandb version 0.13.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-08-05 14:31:25,091 INFO    MainThread:12868 [wandb_init.py:init():700] starting run threads in backend
2022-08-05 14:31:30,111 INFO    MainThread:12868 [wandb_run.py:_console_start():1893] atexit reg
2022-08-05 14:31:30,112 INFO    MainThread:12868 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.WRAP
2022-08-05 14:31:30,112 INFO    MainThread:12868 [wandb_run.py:_redirect():1803] Wrapping output streams.
2022-08-05 14:31:30,114 INFO    MainThread:12868 [wandb_run.py:_redirect():1827] Redirects installed.
2022-08-05 14:31:30,114 INFO    MainThread:12868 [wandb_init.py:init():732] run started, returning control to user process
2022-08-05 14:31:30,114 INFO    MainThread:12868 [wandb_config.py:__setitem__():150] config set batch_size = 256 - <bound method Run._config_callback of <wandb.sdk.wandb_run.Run object at 0x000002144CE74048>>
2022-08-05 14:31:30,115 INFO    MainThread:12868 [wandb_run.py:_config_callback():1131] config_cb batch_size 256 None
2022-08-05 14:31:32,078 INFO    MainThread:12868 [wandb_run.py:_tensorboard_callback():1239] tensorboard callback: experiments-pushing\0\0\model\SAC_1, True
2022-08-05 14:45:26,343 INFO    MainThread:12868 [wandb_init.py:_pause_backend():374] pausing backend
2022-08-05 14:45:26,343 INFO    MainThread:12868 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-08-05 14:48:29,210 INFO    MainThread:12868 [wandb_init.py:_resume_backend():383] resuming backend
2022-08-05 14:48:29,238 INFO    MainThread:12868 [wandb_setup.py:_flush():76] Configure stats pid to 12868
2022-08-05 14:48:29,238 INFO    MainThread:12868 [wandb_setup.py:_flush():76] Loading settings from C:\Users\psiml8\.config\wandb\settings
2022-08-05 14:48:29,238 INFO    MainThread:12868 [wandb_setup.py:_flush():76] Loading settings from c:\Users\psiml8\CausalWorld\scripts\wandb\settings
2022-08-05 14:48:29,238 INFO    MainThread:12868 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-08-05 14:48:29,239 INFO    MainThread:12868 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-08-05 14:48:29,239 INFO    MainThread:12868 [wandb_init.py:_log_setup():461] Logging user logs to c:\Users\psiml8\CausalWorld\scripts\wandb\run-20220805_144829-1euce511\logs\debug.log
2022-08-05 14:48:29,240 INFO    MainThread:12868 [wandb_init.py:_log_setup():462] Logging internal logs to c:\Users\psiml8\CausalWorld\scripts\wandb\run-20220805_144829-1euce511\logs\debug-internal.log
2022-08-05 14:48:29,240 INFO    MainThread:12868 [wandb_init.py:init():495] calling init triggers
2022-08-05 14:48:29,240 INFO    MainThread:12868 [wandb_init.py:init():499] wandb.init called with sweep_config: {}
config: {}
2022-08-05 14:48:29,241 INFO    MainThread:12868 [wandb_init.py:init():515] re-initializing run, found existing run on stack: 16s1mhi1
2022-08-05 14:48:29,243 INFO    MainThread:12868 [wandb_run.py:_finish():1680] finishing run natasa-jovanovic/causal-world/16s1mhi1
2022-08-05 14:48:29,244 INFO    MainThread:12868 [jupyter.py:save_history():435] not saving jupyter history
2022-08-05 14:48:29,244 INFO    MainThread:12868 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2022-08-05 14:48:29,245 INFO    MainThread:12868 [wandb_init.py:_jupyter_teardown():393] cleaning up jupyter logic
2022-08-05 14:48:29,245 INFO    MainThread:12868 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 0
2022-08-05 14:48:29,247 INFO    MainThread:12868 [wandb_run.py:_restore():1834] restore
2022-08-05 14:48:29,551 INFO    MainThread:12868 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
  other_count: 1
}
pusher_stats {
  uploaded_bytes: 9708966
  total_bytes: 9708966
}

2022-08-05 14:48:34,679 INFO    MainThread:12868 [wandb_run.py:_on_finish():1991] got exit ret: None
